libs/elasticsearch/langchain_elasticsearch/vectorstores.py

```diff
@@ -680,6 +329,10 @@ def similarity_search(
         k: int = 4,
         fetch_k: int = 50,
         filter: Optional[List[dict]] = None,
+        custom_query: Optional[
```

Comment:
```suggestion
        *,
        custom_query: Optional[
```
make these kwarg-only to make modifying in future easier.